### PR TITLE
Version v11.7.2

### PR DESCRIPTION
**Pull request by danjm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.7.2]
+### Added
```
Comment (Gudahtt) [inline CHANGELOG.md:10]: This changelog needs to be fixed; we have two empty sections and we're missing a change entry

File: privacy-snapshot.json [hunk 2]
```diff
@@ -38,5 +38,6 @@
   "token-api.metaswap.codefi.network",
   "tx-insights.metaswap.codefi.network",
   "unresponsive-rpc.url",
-  "www.4byte.directory"
+  "www.4byte.directory",
+  "price-api.metafi.codefi.network"
```
Comment (Gudahtt) [inline privacy-snapshot.json:42]: We should be able to remove `api.coingecko.com` from this list now as well

File: app/scripts/metamask-controller.js [hunk 3]
```diff
@@ -571,7 +565,7 @@ export default class MetamaskController extends EventEmitter {
       onPreferencesStateChange: this.preferencesController.store.subscribe.bind(
         this.preferencesController.store,
       ),
-      onNetworkStateChange: networkControllerMessenger.subscribe.bind(
+      onNetworkDidChange: networkControllerMessenger.subscribe.bind(
         networkControllerMessenger,
         'NetworkController:stateChange',
```
Comment (Gudahtt) [inline app/scripts/metamask-controller.js:570]: We should be subscribed to `networkDidChange` here

File: app/scripts/metamask-controller.js [hunk 4]
```diff
@@ -571,7 +565,7 @@ export default class MetamaskController extends EventEmitter {
...
       ),
```
Comment (Gudahtt) [inline app/scripts/metamask-controller.js:571]: A few lines below here, the `getERC20TokenName` option should be removed. This option is not accepted as of v21

>BREAKING: TokensController.watchAsset now performs on-chain validation of the asset's symbol and decimals, if they're defined in the contract (https://github.com/MetaMask/core/pull/1745)
>
>    The TokensController constructor no longer accepts a getERC20TokenName option. It was no longer needed due to this change.

**Comment by Gudahtt** [comment]
I found a few minor issues with the code changes here, but we should fix them on `develop` first before fixing them here. Any changes introduced in this PR will be undone with the sync back to `develop`.

File: CHANGELOG.md (hunk 1 above)
Comment (DDDDDanica) [reply CHANGELOG.md]: Hey @Gudahtt i believe this list is still WIP but i have included the latest changes here [#1a6d367a](https://github.com/MetaMask/metamask-extension/pull/22336/commits/1a6d367a298eff2ea6d18381c2cbc74b82fced12)  please take a look at it and let me know if there's any issues.

File: app/scripts/metamask-controller.js (hunk 3 above)
Comment (danjm) [reply app/scripts/metamask-controller.js]: @Gudahtt The code changes you commented on are now part of v11.7.3 https://github.com/MetaMask/metamask-extension/pull/22354

I wanted to clarify on the point you made here though. Are you saying these lines of code should be changed to:
```
onNetworkDidChange: networkControllerMessenger.subscribe.bind(
    networkControllerMessenger,
    'NetworkController:networkDidChange',
)
```

**Comment by danjm** [comment]
The commits in this PR have changed from when it was first cut. What was v11.7.2 yesterday is now v11.7.3 https://github.com/MetaMask/metamask-extension/pull/22354

This new v11.7.2 contains a single commit focused on fixing transaction status display bugs

**Comment by sleepytanya** [comment]
Confirmed for all test networks on Chrome and Firefox:

1. incoming transactions history on Linea Goerli is fixed https://github.com/MetaMask/metamask-extension/issues/22237
2. saved gas fee (Advanced gas) applied to all subsequent transactions unless user changes gas settings to another option https://github.com/MetaMask/metamask-extension/issues/22333
3. I don't see 'dropped' transaction issue (scenario 1 - create tx, switch network, create tx with the same account and with the same nonce, scenario 2 - send eth between two accounts on the same wallet and same network with the same nonce / higher / lower nonce) https://github.com/MetaMask/metamask-extension/issues/22312
4. I sent around 50 txs while testing but encountered 'pending' transaction only once and the status was updated pretty quick so we can count it as accidental one-time issue (?) and consider 'pending' transaction issue fixed https://github.com/MetaMask/metamask-extension/issues/21941

File: privacy-snapshot.json (hunk 2 above)
Comment (danjm) [reply privacy-snapshot.json]: Fixed here https://github.com/MetaMask/metamask-extension/pull/22372

File: app/scripts/metamask-controller.js (hunk 3 above)
Comment (danjm) [reply app/scripts/metamask-controller.js]: Fixed here https://github.com/MetaMask/metamask-extension/pull/22372

File: app/scripts/metamask-controller.js (hunk 4 above)
Comment (danjm) [reply app/scripts/metamask-controller.js]: Fixed here https://github.com/MetaMask/metamask-extension/pull/22372

**Pull request merged**